docs/features/tenancy.md

```diff
@@ -83,39 +97,40 @@ f15c1f6d-5df5-4a00-8f20-77c8e7a7af11  Product1Test
 Lets limit dev team to create volumes of total 20 Gb each not exceeding size of 1 GB. Similarly for QA team, lets put restriction of total 40 GB consumption with each volume size not exceeding 1 GB.
 
 ```
-#/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access add --name=Product1Dev --datastore=datastore3  --allow-create --default-datastore --volume-maxsize=1GB --volume-totalsize=20GB
-vmgroup access add succeeded
+[root@sc-rdops-vm17-dhcp-3-236:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access set --name=Product1Dev --datastore=sharedVmfs-0  --volume-maxsize=1GB --volume-totalsize=20GB
+vmgroup access set succeeded
+
+[root@sc-rdops-vm17-dhcp-3-236:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access set --name=Product1Test --datastore=sharedVmfs-0  --volume-maxsize=1GB --volume-totalsize=40GB
+vmgroup access set succeeded
 
-#/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access add --name=Product1Test --datastore=datastore3  --allow-create --default-datastore --volume-maxsize=1GB --volume-totalsize=40GB
-vmgroup access add succeeded
 ```
 Lets verify that storage restrictions has been set properly.
```

Comment:
not for this review, as you are here to update .. can you address following nit?

`restrictions has been set properly` => `restrictions have been set properly`